packages/react-native-otlp/test-project/android/settings.gradle

```diff
@@ -0,0 +1,45 @@
+import org.gradle.api.initialization.resolve.RepositoriesMode
+
+pluginManagement {
+    repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
+        gradlePluginPortal()
+    }
+
+    // including external React Native plugin (from root of monorepo)
+    // TBD if the absolute path could be calculated dynamically
+    includeBuild("../node_modules/@react-native/gradle-plugin")
+}
+
+// React configuration (needs 0.75.x version)
+plugins {
+    id 'com.facebook.react.settings'
+}
+
+// React configuration (needs 0.75.x version)
+extensions.configure(com.facebook.react.ReactSettingsExtension) {
+    ex -> ex.autolinkLibrariesFromCommand()
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.PREFER_PROJECT)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+// using local package
+include ':react-native-otlp'
```

Comment:
@jpmunz this (and next line) is also part of what we need to connect the module into the test-project without copying/pasting src files

Reply:
oh nice! Ya this is definitely a cleaner approach than the copying